packages/grid/_modules_/grid/components/panel/filterPanel/FilterInputValue.tsx

```diff
@@ -39,7 +39,7 @@ export function FilterInputValue(props: TypeFilterInputValueProps) {
     setFilterValueState(item.value || '');
   }, [item.value]);
 
-  const inputProps = applying ? { endAdornment: <LoadIcon /> } : undefined;
+  const inputProps = applying ? { endAdornment: <LoadIcon /> } : others['inputProps'];
```

Comment:
It's used as `InputProps` not `inputProps`.

```suggestion
  const InputProps = applying ? { endAdornment: <LoadIcon /> } : others.InputProps;
```